exercicios/index.html

```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
     <main>
-        <h1>Exercícios de HTML5 e CSS3</h1>
+        <h1>Exercícios de HTML5 et CSS3</h1>
```

Comment:
Acho que não existe motivos para essa modificação 🤔
```suggestion
        <h1>Exercícios de HTML5 e CSS3</h1>
```